react/hooks/useStartBulkImport.ts

```diff
@@ -0,0 +1,14 @@
+import useSWRMutation from 'swr/mutation'
+
+import { startBulkImport } from '../services'
+
+const useStartBulkImport = () => {
+  const { trigger, ...data } = useSWRMutation(
+    '/buyer-orgs',
+    (_, { arg }: { arg: string }) => startBulkImport(arg)
+  )
+
+  return { startBulkImport: trigger, ...data }
+}
```

Comment:
The hook useStartBulkImport is responsible to manage the importation of file, the argument is the import Id